src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -88,6 +89,41 @@ ss::future<response_ptr> describe_configs_handler::handle(
               topic_config->replication_factor,
               describe_configs_source::topic);
 
+            ss::sstring cleanup_policy;
```

Comment:
nit: could be a free function really. Given a topic_config, return the cleanup policy.

Reply:
fixed